huff_cli/src/huffc.rs

```diff
@@ -82,6 +82,10 @@ struct Huff {
     #[clap(short = 'v', long = "verbose")]
     verbose: bool,
 
+    /// Prints out the jump label PC indices for the specified contract.
+    #[clap(short = 'x', long = "label-indices")]
```

Comment:
hmm i see that `-l` is taken by alt constructor. `-l` suits this one waaaaay better. Maybe swap alt-constructor to `-t`? Open to suggestions though its not a widely used feature.

Reply:
changed it to `-l` as suggested for now